docs/resources/gateway.md

```diff
@@ -501,25 +501,29 @@ tags = {
 * `gateway_gwlb_endpoints` - (AWS, Azure) For AWS, this attribute includes the Gateway Load Balancer (GWLB) endpoints created in each of the AZs displayed in the format as follows:
 
     ```hcl
-    gateway_gwlb_endpoints {
-        availability_zone    = "us-east-1a"
-        endpoint_id          = "vpce-047c749fc6f7e0c0d"
-        network_interface_id = "eni-017eacdb23d2ebaf4"
-        subnet_id            = "subnet-0d61750e97caafd9d"
-    }
-    gateway_gwlb_endpoints {
-        availability_zone    = "us-east-1b"
-        endpoint_id          = "vpce-0707fa3f03c5064a7"
-        network_interface_id = "eni-020464bd838461bca"
-        subnet_id            = "subnet-0fd61e07f200224f1"
-    }
+    gateway_gwlb_endpoints: [

```

Comment:
why was this changed like this? I thought the output is printed like what we had in the old code?

Reply:
May be convert this to proper json